crates/oxc_parser/src/lib.rs

```diff
@@ -330,6 +330,34 @@ mod test {
         assert_eq!(ret.errors.first().unwrap().to_string(), "Source length exceeds 4 GiB limit");
     }
 
+    #[test]
```

Comment:
Add then here: https://github.com/oxc-project/oxc/tree/main/tasks/coverage/misc/pass

We don't need to bloat this file with syntax tests

Reply:
That's what I want to know